options/posix/include/termios.h

```diff
@@ -25,6 +25,8 @@ typedef unsigned int tcflag_t;
 #define VSTOP    8
 #define VSUSP    9
 #define VTIME   10
+#define VLNEXT  15
+#define VEOL2   16
```

Comment:
This would need an adjustment of NCCS which is an ABI break (because it changes the size of `struct termios`). It is okay to break managarm's ABI if we need this but we should move these definitions and the struct into an `abi-bits/` header and keep the old definitions for all other ports.